backend/compact-connect/lambdas/common-python/cc_common/data_model/client.py

```diff
@@ -196,6 +196,7 @@ def create_privilege(self, *, compact: str, jurisdiction: str, provider_id: str)
                                     'status': provider_data['status'],
```

Comment:
Accessibility: This form does not contain a submit button, which creates issues for those who cannot submit the form using the keyboard. Submit buttons are INPUT elements with type attribute "submit" or "image", or BUTTON elements with type "submit" or omitted/invalid.

@jsandoval81 

<img width="422" alt="image" src="https://github.com/user-attachments/assets/9202f4a5-3cb2-41f6-aaa4-6df04f4aa9fd">